### DISCUSSION TITLE
singleton support for international | singleton with two document writer

### DISCUSSION THREAD

**Discussion by TajAlasfiyaa** [Thinkmill/keystatic#813] [Ideas]
![image](https://github.com/Thinkmill/keystatic/assets/87016999/d92b4b1b-5648-44d2-9bb4-07a64ef3c591)
i am devopling multi language website and i want the localiztion for home page to be in one file

**Comment by TajAlasfiyaa** [comment]
i mean the ui look weird

**Comment by simonswiss** [comment]
Hey!

I imagine your `home` singleton has the `entryLayout: 'content'` option in the Keystatic config.

If you need two document fields, you probably want to opt out of that `entryLayout: 'content'` and use the default one (`form`) where all fields are statcked on top of each other.

Deleting the `entryLayout`option in your config will do the trick, since `form` is the default.

Hope it helps!
